CHANGES.rst

```diff
@@ -25,6 +25,8 @@ Improvements
   restrict them to speakers or registrants (:pr:`5670`, thanks :user:`kewisch`)
 - Make editing timeline display much more straightforward (:pr:`5674`)
 - Allow event managers to delete editables from contributions (:pr:`5778`)
+- Support custom user-defined stylesheets with text metric overrides (:pr:`5895`)
+- Add ``<ind-menu>`` custom element and make the header toolbar dropdown accessible (:pr:`5897`)
```

Comment:
I think this better fits into "Internal Changes" (and maybe we can consider introducing an a11y section later)

Also, credit yourself ;) see the "thanks :user:...` in some other places for contributions coming from outside the core indico team